Makefile

```diff
@@ -22,6 +22,12 @@ DEV_IMG ?= # <== NOTE:  outside dev, change this!!!
 VERSION ?= $(shell git describe --exact-match 2> /dev/null || \
 	   git describe --match=$(git rev-parse --short=8 HEAD) --always --dirty --abbrev=8)
 
+KUSTOMIZE_VERSION = $(kustomize version | awk '{ print $2 }' | awk -F ':' '{ print $2} ')
```

Comment:
Which shell are you using?  On my Mac, I'm using bash and when I tried this code in the makefile, it doesn't work.  The variable is empty upon executing the above.  If I execute the command at a terminal, it works fine.

Reply:
It appears you want
```
KUSTOMIZE_VERSION ?= $(shell kustomize version | awk '{ print $$2 }' | awk -F ':' '{ print $$2} ')
```

Reply:
Yes, I found the error, but I use the ':=' because KUSTOMIZATION_VERSION is not a environment variable.